BusyBox pidof does not support '-c' yet (checked v1.35.0)
For our purposes we don't need -c, if any mstpd runs is enough info.

--- mstpd-0.1.0/bridge-stp.in.orig	2022-02-04 11:49:59.764280663 +0100
+++ mstpd-0.1.0/bridge-stp.in	2022-02-04 11:51:00.212347703 +0100
@@ -139,7 +139,7 @@
         fi
 
         # Start mstpd if necessary.
-        if ! pidof -c -s mstpd >/dev/null; then
+        if ! pidof -s mstpd >/dev/null; then
             if [ "$MANAGE_MSTPD" != 'y' ]; then
                 errmsg 'mstpd is not running'
                 exit 3
@@ -212,12 +212,12 @@
         done
 
         # Kill mstpd, since no bridges are currently using it.
-        kill $(pidof -c mstpd)
+        kill $(pidof mstpd)
         ;;
     restart|restart_config)
         if [ "$action" = 'restart' ]; then
             # Kill mstpd.
-            pids="$(pidof -c mstpd)" ; Err=$?
+            pids="$(pidof mstpd)" ; Err=$?
             if [ $Err -eq 0 ]; then
                 echo 'Stopping mstpd ...'
                 kill $pids
